corgi/collectors/errata_tool.py

```diff
@@ -158,41 +158,38 @@ def get_erratum_components(self, erratum_id: str):
 
         return dict(variant_to_component_map)
 
-    # TODO update this to use the ET Models
-    def variant_cdn_repo_mapping(self) -> dict:
+    def save_variant_cdn_repo_mapping(self):
         """Fetches all existing Errata Tool Variants and the CDN repos they are configured with.
 
         Each erratum's builds are mapped against individual Variants that determine which repos
         the content gets pushed to.
         """
-        variants = self.get_paged("api/v1/variants", page_data_attr="data")
-
-        # Index a list of variants by their name so we can add repos to them, and pull out only
-        # data that we need out of them.
-        variants_with_repos = {}
-        for variant in variants:
-            variant_attrs = variant["attributes"]
-            variants_with_repos[variant_attrs["name"]] = {
-                "description": variant_attrs["description"],
-                "cpe": variant_attrs["cpe"],
-                "repos": [],
-            }
 
         for repo in self.get_paged("api/v1/cdn_repos", page_data_attr="data"):
             repo_attrs = repo["attributes"]
             repo_rels = repo["relationships"]
 
             for variant in repo_rels["variants"]:
                 variant_name = variant["name"]
-                if variant_name not in variants_with_repos:
-                    logger.warning(
+                try:
+                    variant_obj = CollectorErrataProductVariant.objects.get(name=variant_name)
+                    variant_obj.repos.append(repo_attrs["name"])
+                except CollectorErrataProductVariant.DoesNotExist:
+                    logger.debug(
```

Comment:
Do we want this to be a debug statement only? I think it indicates some faulty data so should probably be reported as an error or a warning.

Reply:
I will update it to warning.